packages/arb-token-bridge-ui/tests/support/commands.ts

```diff
@@ -36,6 +40,41 @@ export const logout = () => {
   })
 }
 
+export const sendEthToAccount = (
+  accountNameOrNumberFrom: string | number,
+  accountNameOrNumberTo: string | number,
+  amount: number
+) => {
+  cy.switchMetamaskAccount(accountNameOrNumberTo)
+  cy.getMetamaskWalletAddress().then(async address => {
+    cy.switchMetamaskAccount(accountNameOrNumberFrom)
+    const wallet = getWallet()
+    const tx = {
+      to: address,
+      value: utils.parseEther(String(amount))
+    }
+    await wallet.sendTransaction(tx)
+  })
+}
```

Comment:
Couple of things here:
1. Since we're always gonna be sending the ETH from the wallet we have in `getWallet`, we can omit that argument
2. We should be explicit about the address we are sending to, and not use the index number 
3. We should be able to specify a provider so we can do it on both networks
4. Can we make the amount be `BigNumber`
5. Why do we have to use Cypress utils for this? We should be able to just send the tx in the background as we do for `wrapEth`

```suggestion
export const sendEth = async (
  to: string,
  amount: number,
  provider: Provider
) => {
  const wallet = getWallet(provider)
  const from = await wallet.getAddress()
  
  const tx = await wallet.sendTransaction({ to, value: amount })
  await tx.wait();
}
```